packages/rocketchat-livechat/app/client/stylesheets/main.less

```diff
@@ -30,6 +30,169 @@ body {
 	border-top-left-radius: 5px;
 }
 
+.mobile-view{
```

Comment:
Why did you revert from using `@media`?

Reply:
The media-querys does not work with Iframe elements, because for the Query the Iframe has always a widht of 320px. This means you can not work with the width of the Window. The only alternetiv is, to check the User-Agent.

Reply:
@HenrikScholl Do you have a qualified web reference that explains the exact issue, you encountered here? I found several different possible options that can cause that problem and for some of them there are alternatives available without using jquery. E.g. https://stackoverflow.com/questions/27227214/media-queries-not-working-inside-an-iframe?rq=1